segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -47,6 +47,8 @@
     protected static final String APPEND_FORMAT = "Segment_%s_Append_%d";
     private static final int SEGMENT_COUNT = 4;
 
+    protected boolean isTestingSystemSegment = false;
```

Comment:
It is bad practice to create protected non-final fields. Please make this private and provide whatever access you need to it via protected methods.

Reply:
fixed